### ISSUE TITLE
bug(pdf): incorrect selected area

### ISSUE THREAD

**Issue by Kristinita** [UB-Mannheim/tesseract#20]
### 1. Possibly related issue

[**#1712**](https://github.com/tesseract-ocr/tesseract/issues/1712).

### 2. Summary

PDF viewers incorrect select words from PDF, that create by Tesseract.

### 3. Data

Example files from my book:

+ [**`KiraProcessedTIF.tif`**](https://app.box.com/s/9fun2fuhst8mtkhv7ha5axjzospdgykk) — TIF image
+ [**`KiraSuperhero.pdf`**](https://app.box.com/s/ln4l2vg42inhp94oyhr3oui3cyvzdobi) — PDF, that create Tesseract
+ [**`KiraCorrectOCR.pdf`**](https://app.box.com/s/ah7jo1qkpsddzv5myd16vg122fytj8wd) — PDF with correct OCR for comparing

### 4. Steps to rperoduce

I download 64-bit Windows version from [**here**](https://github.com/UB-Mannheim/tesseract/wiki#tesseract-at-ub-mannheim), how described [**in official Tesseract wiki**](https://github.com/tesseract-ocr/tesseract/wiki#windows) → in installation process I select Russian (`rus`) additional language → I install Tesseract → I add path with `tesseract.exe` as user `PATH` environment variable → I run command:

```text
tesseract KiraProcessedTIF.tif KiraSuperhero -l rus pdf
```

### 5. Expected behavior

For `KiraCorrectOCR` text select correctly in any program:

![KiraCorrectOCR Марк](https://i.imgur.com/sE5Y1CJ.png)

![KiraCorrectOCR самоосмысление](https://i.imgur.com/VhoSN6U.png)

### 6. Actual behavior

For `KiraSuperhero` Tesseract select not full word:

![KiraSuperhero Марк](https://i.imgur.com/UKZhM4A.png)

![KiraSuperhero самоосмысление](https://i.imgur.com/VtGvX1b.png)

It reproduced for any word in `KiraSuperhero`.

### 7. Not helped

I reproduce actual behavior for `KiraSuperhero` in any PDF viewer.

+ Firefox:

![Firefox](https://i.imgur.com/6FZTvGS.png)

+ PDF-XChange Editor:

![PDF-XChange](https://i.imgur.com/TY9QBls.png)

### 8. Environment

+ Windows 10 Enterprise LTSB 64-bit EN

```text
D:\SashaDebugging\KiraGoddess>tesseract --version
tesseract v5.0.0-alpha.20190708
 leptonica-1.78.0
  libgif 5.1.4 : libjpeg 8d (libjpeg-turbo 1.5.3) : libpng 1.6.34 : libtiff 4.0.9 : zlib 1.2.11 : libwebp 0.6.1 : libopenjp2 2.3.0
 Found AVX2
 Found AVX
 Found SSE
 Found libarchive 3.3.2 zlib/1.2.11 liblzma/5.2.3 bz2lib/1.0.6 liblz4/1.7.5
```

Thanks.

**Comment by stweil** [comment]
Yes, the positions are not exact. That also happens with the old OCR engine (`--psm 0`).

**Comment by stweil** [comment]
I am afraid there is no fast solution. It looks like this problem is already rather old.

**Comment by Kristinita** [comment]
@stweil , **Type: Question ❓**

If this issue is «duplicate», can you show an issue, regarding which my issue is duplicate?

I want to subscribe to progress for solving on this issue.

Thanks.

**Comment by stweil** [comment]
It is a duplicate of https://github.com/tesseract-ocr/tesseract/issues/1712.
